src/manager/msWindowManager.ts

```diff
@@ -63,6 +63,8 @@ const INF_COST = 100000;
  */
 const MAX_WINDOW_REASSOCIATION_TIME_MS = 3000;
 
+const CREATED_LESS_THAN = 2000;
```

Comment:
That's an absolutely terrible name! Sorry, but it is.

```suggestion
/** Windows that are created, but then have their metawindow removed from them very quickly are not desirable.
 * So we kill those windows when that happens.
 */
const WINDOW_RECENTLY_CREATED_TIME_MS = 2000;
```

Reply:
Your hurt my feeling ! :laughing: